packages/components/src/components/common/table/hooks/hooks.tsx

```diff
@@ -175,6 +177,7 @@ export const useTableKeyHandler = <TData,>({ table, data, options }: TableKeyboa
       table.resetRowSelection();
       allRows[newReorderIndex].toggleSelected();
       setRootIndex(newReorderIndex);
+      event.currentTarget.rows[newReorderIndex].scrollIntoView({ block: 'center' });
```

Comment:
no negative effect if no scroll bar?

Reply:
I have tested it a little and I have not noticed any. If you like, you are welcome to try it out a little and check if you notice something, but I don't think there should be any negative effects, as the arrows within a table don't have any effect on a table without a scrollbar.